jupyter_server/extension/application.py

```diff
@@ -205,6 +206,14 @@ def static_url_prefix(self):
         help=_("""Handlers appended to the server.""")
     ).tag(config=True)
 
+    open_browser = Bool(False,
```

Comment:
drive by comment but it looks like the previous behavior was to always open_browser, and here you're changing the default to not open the browser. Did you want to retain the previous behavior by default, or at least document the change if the new default behavior is to not open on purpose?

Reply:
Yeah, True by default makes more sense. I have pushed that change. We can keep this comment open before merging in case anyone has a different opinion on this.